tools/snpeff.py

```diff
@@ -59,7 +59,7 @@ def execute(self, command, args, JVMmemory=None, stdin=None, stdout=None):    #
             ] + args
 
         _log.debug(' '.join(tool_cmd))
-        return util.misc.run_and_print(tool_cmd, stdin=stdin, buffered=True, silent=("databases" in command), check=True)
+        return util.misc.run_and_print(tool_cmd, stdin=stdin, stderr=stderr, buffered=True, silent=("databases" in command), check=True)
```

Comment:
Any reason we can just revert this to subprocess.check_call?

Reply:
We probably should revert. Let's save it for a separate PR? This one is already larger than it should be.